modules/node_modules/@ncigdc/components/Case.js

```diff
@@ -156,8 +164,16 @@ const Case = ({
     });
   }, []);
 
+  const filesToAdd = p.files.filter(f => !files.some(cf => cf.file_id === f.file_id));
+
   return (
     <Column spacing={theme.spacing}>
+      <Row style={{ justifyContent: 'flex-end' }}>
+        <Button onClick={() => dispatch(toggleInCart(filesToAdd.length ? filesToAdd : p.files))}>
```

Comment:
I get 
```Uncaught TypeError: (0 , _cart.toggleInCart) is not a function
    at onClick (http://localhost:8080/js/bundle.js:148054:52)
```
on this button, plus I don't see it in `dux/cart.js` maybe it got renamed to `toggleFilesInCart` behind your back